cl/search/templates/search.html

```diff
@@ -218,6 +217,19 @@
             </div>
         </div>
 
+        {% if search_form.type.value == SEARCH_TYPES.OPINION and missing_citations %}
+          <div class="alert alert-info" role="alert">
+            <div class="row">
+                <div class="col-xs-12">
+                  <p id="missing-citations"><i class="fa fa-info-circle fa-lg"></i>
+                    Showing results for <strong>"{{ suggested_query }}"</strong> without citation{{ missing_citations|pluralize }}
+                    {% for citation in missing_citations %} <strong>"{{ citation }}"</strong>{% if not forloop.last %}, {% endif %}{% endfor %}.</br>
+                    It appears we don't yet have that citation{{ missing_citations|pluralize }}.
```

Comment:
A couple tweaks:

```suggestion
                    Showing results for <strong>"{{ suggested_query }}"</strong> without citation{{ missing_citations|pluralize }}
                    {% for citation in missing_citations %} <strong>"{{ citation }}"</strong>{% if not forloop.last %}, {% endif %}{% endfor %}. It appears we don't yet have {{ missing_citations|pluralise:"that citation,those citations" }}.
```

(fixes pluralize and I don't think we need the br.)

Reply:
Thanks pluralize fixed:

![Screenshot 2024-08-27 at 4 36 41 p m](https://github.com/user-attachments/assets/658c8bd4-9303-4e3e-8665-d6cb0e4179e2)
![Screenshot 2024-08-27 at 4 35 37 p m](https://github.com/user-attachments/assets/461bfa3c-e500-46a0-9d6d-419985d9467c)